File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.*;
class Solution {
    public int[] solution(int l, int r) {
        List<Integer> list = new ArrayList<>();
        
        for(int i=l; i<=r; i++) {
            String[] arr = Integer.toString(i).split("");
            boolean flag = true;
            for(int j=0; j<arr.length; j++) {
                if(!arr[j].equals("0") && !arr[j].equals("5")) {
                    flag = false;
                    break;
                }
            }
            if(flag) list.add(i);
        }
        
        if(list.size() == 0) {
            int[] a = {-1};
            return a;
        }
            
        int[] answer = new int[list.size()];
        for(int i=0; i<answer.length; i++) {
            answer[i] = list.get(i);
        }
        return answer;
    }
}

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    
    for i in range(l, r+1) :
        if isPossible(i) :
            answer.append(i)
    
    if len(answer) == 0 :
        return [-1]
    return answer

def isPossible(i) :
    for s in str(i) :
        if s != '0' and s != '5' :
            return False
    return True